README.md

```diff
@@ -142,6 +142,8 @@ We have Element channels for real-time discussions on Web3 and Polkadot. Join th
 
 ## :rocket: Alternative Funding Sources
 
+Some funding sources might be more and some less suitable for your project throughout its various stages. We encourage you to explore alternative funding options listed below. Please note, however, that you should not seek to fund the **same scope of work** from multiple sources and that any team found doing so will have its Web3 Foundation support terminated.
```

Comment:
```suggestion
Some funding sources might be more and some less suitable for your project throughout its various stages. We encourage you to explore alternative funding options listed below. Please note, however, that you should not seek to fund the **same scope of work** from multiple sources and that any team found doing so will have its Web3 Foundation grant terminated.
```

Reply:
I liked support since it implies that all sources will be cut off. And the talk was about funding in general in that paragraph, not necessarily of the grant